service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/mapper/AddressMapper.java

```diff
@@ -0,0 +1,54 @@
+package uk.nhs.adaptors.oneoneone.cda.report.mapper;
+
+import java.util.Arrays;
+import uk.nhs.adaptors.oneoneone.cda.report.util.NodeUtil;
+import uk.nhs.connect.iucds.cda.ucr.AD;
+
+import org.hl7.fhir.dstu3.model.Address;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class AddressMapper {
+
+    @Autowired
+    PeriodMapper periodMapper;
```

Comment:
See Java coding standards. Let's try to use constructors instead of field injection.
Field should be private